README.md

```diff
@@ -1,16 +1,16 @@
 # Description
 
-Switches workspaces in the [i3 window manager](https://i3wm.org/) like xmonad.
-This script will always bring the requested workspace to the current monitor
-("output" in i3's terminology). If the requested workspace is showing on another
-monitor, then the current workspace and requested workspace will swap positions
-with each other.
+Switches workspaces in the [i3 window manager](https://i3wm.org/) and
+[sway](https://swaywm.org/) like xmonad. This script will always bring the
+requested workspace to the current monitor ("output" in i3's terminology). If
+the requested workspace is showing on another monitor, then the current
+workspace and requested workspace will swap positions with each other.
 
 
 # Requirements
 
 - [i3 window manager](https://i3wm.org/)
-- [i3-py](https://github.com/ziberna/i3-py)
+- [i3ipc](https://github.com/acrisci/i3ipc-python)
```

Comment:
1. Please describe this as "i3ipc python module"
2. Add a sub bullet "Install with `pip install i3ipc`".